modules/token/client/cli/tx.go

```diff
@@ -106,7 +106,7 @@ func GetCmdIssueToken() *cobra.Command {
 				return err
 			}
 
-			var prompt = "The token issue transaction will consume extra fee"
+			var prompt = "The token issurance transaction will consume extra fee"
```

Comment:
issurance is irrelevant with issue(as I know).

Reply:
typo